### ISSUE TITLE
Add wind_gust and uvi fields to onecall hourly forecasts

### ISSUE THREAD

**Issue by gumshoes** [awesomio/Weather.NET#4]
Hi,

Any chance you could add the fields
uvi
wind_gust
to the onecall hourly forecasts?

Perhaps they did not exist when you created the API, thanks.

```
"hourly": [
{
"dt": 1618261200,
"temp": 11,
"feels_like": 10.37,
"pressure": 1023,
"humidity": 85,
"dew_point": 8.58,
"uvi": 0.09,
"clouds": 100,
"visibility": 10000,
"wind_speed": 1.44,
"wind_deg": 201,
"wind_gust": 2.11,
"weather": [
{
"id": 804,
"main": "Clouds",
"description": "overcast clouds",
"icon": "04d"
}
],
"pop": 0.72
},
```

https://api.openweathermap.org/data/2.5/onecall?lat=35&lon=139&appid=<YOUR KEY>

**Comment by sharpgraf** [comment]
> Hi,
> 
> Any chance you could add the fields
> uvi
> wind_gust
> to the onecall hourly forecasts?
> 
> Perhaps they did not exist when you created the API, thanks.
> 
> ```
> "hourly": [
> {
> "dt": 1618261200,
> "temp": 11,
> "feels_like": 10.37,
> "pressure": 1023,
> "humidity": 85,
> "dew_point": 8.58,
> "uvi": 0.09,
> "clouds": 100,
> "visibility": 10000,
> "wind_speed": 1.44,
> "wind_deg": 201,
> "wind_gust": 2.11,
> "weather": [
> {
> "id": 804,
> "main": "Clouds",
> "description": "overcast clouds",
> "icon": "04d"
> }
> ],
> "pop": 0.72
> },
> ```
> 
> https://api.openweathermap.org/data/2.5/onecall?lat=35&lon=139&appid=

Hi gumshoes ! I will add this in the next update. For now, you can use your own OneCallModel based class simply by adding the fields you want.
```c#
MyOwnOneCallModel dataOneCall = client.GetOneCallApiAsync<MyOwnOneCallModel>(40.12, 96.66, "en", oneCallExcludes).Result;

**Comment by sharpgraf** [comment]
Solved in 1.0.7 version.  Thank you for your message !

**Comment by gumshoes** [comment]
That's great, much appreciated.
